admin/services/data-access/azure-blob.data.service.js

```diff
@@ -9,18 +10,21 @@ if (config.AZURE_STORAGE_CONNECTION_STRING) {
   blobService = azureStorage.createBlobService()
 }
 
-const service = {
-  getBlobText: async (containerName, blobName) => {
+bluebird.promisifyAll(blobService, {
```

Comment:
please thoroughly test this against your own storage account to ensure it works ok.  as it affects all functions on the library

Reply:
@GuyHarwood done against my own storage account and verified it is working as expected.